### DISCUSSION TITLE
fix mobs walking over rails

### DISCUSSION THREAD

**Discussion by Tizooo** [PurpurMC/Purpur#532] [__Completed__] [answered]
by default mobs wont walk over rails. u could use rails to make a wall around a house and zombies/villagers etc. wont walk over it. would be cool to have a option so mobs can normally walk over rails

**Comment by encode42** [comment, marked as answer]
Closed in https://github.com/pl3xgaming/Purpur/commit/635486be4cfe832a3e32ddaa27e88f3f3604a330
